manifests/a/ArchiMateTool/Archi/5.2.0/ArchiMateTool.Archi.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: ArchiMateTool.Archi
+PackageVersion: 5.2.0
+PackageLocale: en-US
+Publisher: Phillip Beauvoir & Jean-Baptiste Sarrodie
+PackageName: Archi
+License: MIT License
+Copyright: Phillip Beauvoir
+ShortDescription: Archi Setup
```

Comment:
Okease provide a better description of what this package is, does, or is used for

Reply:
> Okease provide a better description of what this package is, does, or is used for

Thank you for your comment. I just updated description of Archi.